app/serializers/curator/filestreams/image_serializer.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Curator
+  class Filestreams::ImageSerializer < Filestreams::FileSetSerializer
+    schema_as_json root: :image do
```

Comment:
If we keep this, I'm thinking that the root should still be `file_set`, and we should indicate the subclass by using an attribute `file_set_type`. See comments on `FileSetSerializer`.